tree/treeplayer/inc/ROOT/TTreeProcessorMT.hxx

```diff
@@ -100,6 +103,12 @@ public:
 
    static void SetTasksPerWorkerHint(unsigned int m);
    static unsigned int GetTasksPerWorkerHint();
+
+   void SetEntriesRange(ULong64_t startEntry, ULong64_t endEntry)
```

Comment:
can we do this in `TTreeProcessorMT`'s constructor? dedicated ctor arguments make it harder to forget to call this :smile: